### PR TITLE
New notifier - GitHub Issues

### PR DESCRIPTION
**Pull request by glasnt**
b/229753500

Preview of New GitHub Issues Notifier

This notifier creates an issue against the configured GitHub repo. It also uses the new template feature to allow full customisation of the notification format, including GitHub features like labels and assignees (as shown in the screenshot).

### CODE REVIEW

**Comment by ace-n** [comment]
@prabenzom @Aric1088 hey folks! @glasnt and I were hoping to get this reviewed within the next week or so.

Is that something y'all can arrange? _(Thanks! 🤗 )_

File: githubissues/githubissues.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "title": "Cloud Build [{{.Build.ProjectId}}] {{.Build.BuildTriggerId}}: {{.Build.Status}}",
+    "body": "Cloud Build {{.Build.ProjectId}} {{.Build.BuildTriggerId}} status: **{{.Build.Status}}**\n\n[View Logs]({{.Build.LogUrl}})"
+}
```
Comment (Aric1088) [inline githubissues/githubissues.json:4]: please add a new line

File: githubissues/githubissues.yaml.example [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+apiVersion: cloud-build-notifiers/v1
+kind: GitHubIssuesNotifier
+metadata:
+  name: example-githubissues-notifier
+spec:
+  notification:
+    filter: build.status == Build.Status.FAILURE
+    template: 
+      type: golang
+      uri: gs://project-name/githuissues.json
```
Comment (Aric1088) [inline githubissues/githubissues.yaml.example:24]: fix typo

**Review by Aric1088 — CHANGES_REQUESTED** [review]
just a few small changes, but LGTM

File: githubissues/githubissues.json (hunk 1 above)
Comment (glasnt) [reply githubissues/githubissues.json:4]: Do you mean to the end of this file?

Comment (Aric1088) [reply githubissues/githubissues.json:4]: yep!

**Review by Aric1088 — COMMENTED** [review]
LGTM

**Pull request merged**